quil-rs/src/instruction/classical.rs

```diff
@@ -118,6 +128,8 @@ impl Quil for BinaryOperand {
 }
 
 #[derive(Copy, Clone, Debug, Hash, PartialEq, Eq)]
+#[pyclass(module = "quil.instructions", eq, frozen, hash, get_all)]
+#[pyo3(rename_all = "snake_case")]
```

Comment:
Shouldn't this be `"SCREAMING_SNAKE_CASE"`? I believe that's the Python convention